docs/sdk.md

```diff
@@ -16,33 +16,55 @@ Should you find bugs or in the case you need help please reach out to us in [Dis
 
 To get you started here's how you would instantiate the SDK and fetch some NFTs
 
-1. Install the sdk
+1. Install the sdk and other dependencies
 
 ```shell
-npm install @3rdweb/sdk
+npm install @3rdweb/sdk ethers dotenv

```

Comment:
remove `dotenv` dependency as the sdk supports both client / server uses. dotenv only applies for scripts / server use.

Reply:
True, make sense.